### PR TITLE
news: Add release post for 0.17.1

### PR DESCRIPTION
**Pull request by slotThe**
To get this party started, let us start with a _very rough_ first draft of a release post!  Hopefully inspiration strikes sometime this week.

### CODE REVIEW

File: news/_posts/2022-09-03-xmonad-0-17-1.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    running GHC 9.2 up to version 9.2.3. The issue has been fixed in
+    [GHC 9.2.4] and all later releases.
+
+  + Fixed border color of windows with alpha channel. Now all windows have the
+    same opaque border color.
+
+[GHC bug 21708]: https://gitlab.haskell.org/ghc/ghc/-/issues/21708
+[GHC 9.2.4]: https://discourse.haskell.org/t/ghc-9-2-4-released/4851
+
+## xmonad-contrib 0.17.0
```
Comment (TheMC47) [inline news/_posts/2022-09-03-xmonad-0-17-1.md]: This should be 0.17.1, right?

Comment (eyenx) [inline news/_posts/2022-09-03-xmonad-0-17-1.md]:
```suggestion
## xmonad-contrib 0.17.1
```

**Review by eyenx — CHANGES_REQUESTED** [review]

File: news/_posts/2022-09-03-xmonad-0-17-1.md (hunk 1 above)
Comment (slotThe) [reply news/_posts/2022-09-03-xmonad-0-17-1.md]: indeed!

**Comment by psibi** [comment]
@slotThe What do you think about doing a release of xmonad-extras-0.17.1 also along with it ? There is one module which has been added to it: https://github.com/xmonad/xmonad-extras/pull/32

**Comment by slotThe** [comment]
@psibi yeah that sounds good to me!

File: news/_posts/2022-09-03-xmonad-0-17-1.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    modules may still be found [on the website].
+
+ + `XMonad.Util.NamedScratchpad`: scratchpads are now only based on the
+    argument given to `namedScratchpadManageHook`; all other scratchpad
+    arguments are, while still present, ignored.  Users passing all of
+    their scratchpads to functions like `namedScratchpadAction` (as is
+    shown in the module's documentation) should _not_ notice any
+    difference in X.
+
+  + `behaviourMonad.Util.DynamicScratchpads`: deprecated the module; use
```
Comment (geekosaur) [inline news/_posts/2022-09-03-xmonad-0-17-1.md]: Did `behaviour` belong somewhere else?

**Review by geekosaur — APPROVED** [review]
Aside from that, it looks okay

File: news/_posts/2022-09-03-xmonad-0-17-1.md (hunk 2 above)
Comment (slotThe) [reply news/_posts/2022-09-03-xmonad-0-17-1.md]: hah, indeed :)

**Pull request merged**